src/views/Bond/components/BondList.tsx

```diff
@@ -209,7 +209,7 @@ const BondRow: React.VFC<{ bond: Bond; isInverseBond: boolean }> = ({ bond, isIn
     <TableCell style={{ padding: "8px 0" }}>
       <Link component={NavLink} to={isInverseBond ? `/bonds/inverse/${bond.id}` : `/bonds/${bond.id}`}>
         <TertiaryButton fullWidth disabled={bond.isSoldOut}>
-          {bond.isSoldOut ? "Sold Out" : t`do_bond`}
+          {bond.isSoldOut ? t`Sold Out` : t`Bond`}
```

Comment:
I think `do_bond` is right here so that translators know when bond is a verb rather than a noun

Reply:
I used a different format for the `t` macro and added comments (which will appear in the translation strings file).

The previous approach was resulting in "do_bond" appearing in the interface.